### PR TITLE
Oceanwater 452 use atacama terrain

### PR DESCRIPTION
**Pull request by mogumbo**
There are still shadow artifacts that I am not satisfied with, but those can be extremely difficult to mitigate. I will create a separate ticket for that. Everything else is looking pretty good.

To test this you'll want to use the matching branch in ow_simulator, which puts the lander on the ground.

### CODE REVIEW

File: models/atacama_y1a/materials/scripts/ow_terrain.frag [hunk 1]
```diff
@@ -0,0 +1,261 @@
...
+    vec2( 0.0606228609526, 0.851023996335 )
+  );
+  for (int i = 0; i < 9; i++)
+  {
+    vec4 newUV = uv;
+    newUV.xy += poissonDisk[i] * invShadowMapSize;
+    newUV = newUV / newUV.w;
+    shadow += texture(shadowMap, newUV.xyz);
+  }
+  shadow /= 9.0;
```
Comment (Samahu) [inline models/atacama_y1a/materials/scripts/ow_terrain.frag]: consider declaring the array as const and use the length() property for the loop
```suggestion
  const vec2 poissonDisk[9] = vec2[](
    vec2( 0.0, 0.0 ), 
    vec2( -0.987330103927, 0.127316674408 ), 
    vec2( -0.168435664837, -0.923511462813 ), 
    vec2( 0.637490968702, 0.633257405393 ), 
    vec2( 0.887653811523, -0.295636257708 ), 
    vec2( 0.516231382947, 0.0664456533132 ), 
    vec2( -0.408070991576, -0.332409120252 ), 
    vec2( -0.491072397165, 0.263378713033 ), 
    vec2( 0.0606228609526, 0.851023996335 )
  );
  for (int i = 0; i < poissonDisk.length(); i++)
  {
    vec4 newUV = uv;
    newUV.xy += poissonDisk[i] * invShadowMapSize;
    newUV = newUV / newUV.w;
    shadow += texture(shadowMap, newUV.xyz);
  }
  shadow /= poissonDisk.length();
```

Comment (Samahu) [reply models/atacama_y1a/materials/scripts/ow_terrain.frag]: @mogumbo  on my machine when I made the array const I get a small improvement ~ +1 fps running full screen. Do you see similar behavior on yours?

**Review by Samahu — APPROVED** [review]
The main noticeable problem is the shadows:

image

But since you are addressing this as a separate ticket this can go ahead

File: models/atacama_y1a/materials/scripts/ow_terrain.frag (hunk 1 above)
Comment (mogumbo) [reply models/atacama_y1a/materials/scripts/ow_terrain.frag]: I don't see a performance improvement, but it makes for safer code. Changed in all .frag files.

Comment (Samahu) [reply models/atacama_y1a/materials/scripts/ow_terrain.frag]: Did you try it with **const**?

Comment (mogumbo) [reply models/atacama_y1a/materials/scripts/ow_terrain.frag]: Oops, no. I missed that part. I just tried it and see no performance boost. And I already merged this branch.

**Pull request merged**